bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableIO.java

```diff
@@ -128,6 +137,64 @@ public class CloudBigtableIO {
   @InternalExtensionOnly
   @SuppressWarnings("serial")
   abstract static class AbstractSource extends BoundedSource<Result> {
+
+    private Object writeReplace() {
+      return new SerializationProxy(getConfiguration());
+    }
+
```

Comment:
Can there be a comment to explain the point of SerializationProxy or the methods?